pkgs/by-name/an/ankama-launcher/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  appimageTools,
+  fetchurl,
+  lib,
+}:
+let
+  pname = "ankama-launcher";
+  version = "3.12.17";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://launcher.cdn.ankama.com/installers/production/Ankama%20Launcher-Setup-x86_64.AppImage";
+    hash = "sha256-aD5YYcye/WPpb0Kir55qCL+RGbK8ZSdmi4NKmqKkW/Y=";
+  };
+
+  appimageContents = appimageTools.extract { inherit name src; };
+in
+
+appimageTools.wrapType2 rec {
+  inherit name src;
+  extraPkgs = pkgs: [ pkgs.wine ];
+
+  extraInstallCommands = ''
+    install -m 444 -D ${appimageContents}/zaap.desktop $out/share/applications/ankama-launcher.desktop
+    sed -i 's/.*Exec.*/Exec=ankama-launcher/' $out/share/applications/ankama-launcher.desktop
+    install -m 444 -D ${appimageContents}/zaap.png $out/share/icons/hicolor/256x256/apps/zaap.png
+  '';
```

Comment:
You don't need this extraInstallCommands
```suggestion
```

Reply:
It is not necessary but I prefer to have a desktop file to launch it